frontend/src/app/data/calculation/calculation.actions.ts

```diff
@@ -109,7 +109,17 @@ export const deleteMultipleCalculations = createAction(
 
 export const updateName = createAction(
   '[Calculation] Update calculation name',
-  props<{ index: number, newName: string }>()
+  props<{ calculationId: number, newName: string }>()
+);
+
+export const updateNameSuccess = createAction(
+  '[Calculation] Update calculation name success',
+  props<{ calculationId: number, newName: string }>()
+);
+
+export const renameCalculationFailure = createAction(
```

Comment:
Why not have same naming here as the above one, e.g "UpdateNameFailure" or vice versa?